cli/args/flags.rs

```diff
@@ -12845,22 +12844,17 @@ Usage: deno repl [OPTIONS] [-- [ARGS]...]\n"
       }
     );
 
-    let flags = flags_from_vec(svec![
-      "deno",
-      "run",
-      "--preload",
-      "p1.js,./p2.js",
```

Comment:
Does it still work with multiple preload flags? Ex. `--preload p1.js --preload ./p2.js`?